bertly.py

```diff
@@ -14,6 +14,8 @@
     :license: MIT, see LICENSE for details
 """
 
+import config
```

Comment:
Not sure what the best practice is here - I pulled this out into a `config` module since we need it to be executed before other imports (and my editor's linter didn't seem happy with `load_dotenv` above other imports) but I can see where stickler is coming from.

Reply:
Alright, extracted some config parsing that was happening in the main module into `config` and now it's happy (and I think it makes things read a bit nicer too)! d8c6de5